ui/src/components/Preview/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import Markdown from '@mozilla-frontend-infra/components/Markdown';
+import ListItem from '@material-ui/core/ListItem';
+import Tab from '@material-ui/core/Tab';
+import Tabs from '@material-ui/core/Tabs';
+import TextField from '@material-ui/core/TextField';
+
+export default class Preview extends Component {
+  state = {
+    value: 0,
+  };
+
+  handleChange = (event, value) => {
+    this.setState({ value });
+  };
+
+  render() {
+    const { description, onDescriptionChange, placeholder } = this.props;
```

Comment:
The content of the markdown is not always a "description". We can perhaps change it to `value`. Also please add propTypes : https://github.com/taskcluster/taskcluster/search?q=static+propTypes&unscoped_q=static+propTypes.